src/app/components_ngrx/work-item-quick-preview/work-item-quick-preview.component.ts

```diff
@@ -0,0 +1,418 @@
+    let result: TypeaheadDropdownValue[] = [];
+    let selectedFound: boolean = false;
+    let selectedIterationId;
+    if (this.workItem.iteration && this.workItem.iteration.id) {
+      selectedIterationId = this.workItem.iteration.id;
+    }
+    for (let i=0; i<iterations.length; i++) {
+      result.push({
+        key: iterations[i].id,
+        value: (iterations[i].resolvedParentPath !== '/' ? iterations[i].resolvedParentPath:'') + '/' + iterations[i].name,
+        selected: selectedIterationId === iterations[i].id ? true : false,
+        cssLabelClass: undefined
+      });
+      if (selectedIterationId === iterations[i].id) {
+        selectedFound = true;
+      }
+    };
+    return result;
+  }
+
+  focusArea() {
+    this.iterationSelectbox.close();
+    this.cancelAssignment();
+    this.areas = [
+      ...this.areas,
+      {
+        key: '0',
+        value: '',
+        selected: false,
+        cssLabelClass: 'spinner spinner-sm spinner-inline'
+      }
+    ];
+    this.areas = this.extractAreaKeyValue(this.areasUI);
+  }
+
+  focusIteration() {
+    this.areaSelectbox.close();
+    this.cancelAssignment();
+    this.iterations = [
+      ...this.iterations,
+      {
+        key: '0',
+        value: '',
+        selected: false,
+        cssLabelClass: 'spinner spinner-sm spinner-inline'
+      }
+    ];
+    this.iterations = this.extractIterationKeyValue(this.iterationUI);
+  }
+
+  constructUrl(workItem: WorkItem) {
+    return this.router.url.split('plan')[0] + 'plan/detail/' + workItem.number;
+  }
+
+  onLabelClick(event) {
+    let params = {
+      label: event.attributes.name
+    }
+    // Prepare navigation extra with query params
+    let navigationExtras: NavigationExtras = {
+      queryParams: params
+    };
+
+    // Navigated to filtered view
+    this.router.navigate([], navigationExtras);
+  }
+
+  @HostListener('window:keydown', ['$event'])
+  onKeyEvent(event: any) {
+    event = (event || window.event);
+    // for ESC key handling
+    if (event.keyCode == 27) {
+      try {
+        event.preventDefault(); //Non-IE
+      } catch (x){
+        event.returnValue = false; //IE
+      }
+      if (this.headerEditable) {
+        this.closeHeader();
+      } else if (this.searchAssignee) {
+        this.searchAssignee = false;
+      } else if (this.areaSelectbox && this.areaSelectbox.isOpen()) {
+        this.areaSelectbox.close();
+      } else if (this.iterationSelectbox && this.iterationSelectbox.isOpen()) {
+        this.iterationSelectbox.close();
+    } else {
+        this.closePreview();
+      }
+    }
+  }
+  onOpen(event) {
+    this.onOpenSelector.emit('open');
+  }
+
+  onClose(event) {
+    this.onCloseSelector.emit(cloneDeep(this.selectedLabels));
+  }
+
+  openDropdown() {
+    this.dropdownRef.openDropdown();
+  }
+
+  closeDropdown() {
+    this.dropdownRef.closeDropdown();
+  }
+  closeAddAssignee() {
+    this.activeAddAssignee = false;
+  }
+}
```

Comment:
Delete this file. This file is already deleted in master. we are not using this.